.github/workflows/check_bazel_tests.yml

```diff
@@ -81,6 +81,9 @@ jobs:
         run: |
           bazel test ${{ matrix.additional_args }} ${{ matrix.test }}
         working-directory: ./exaudfclient/
+        env:
```

Comment:
I guess, we didn't need anything else in env, because it wasn't used by java?

Reply:
And, because while I see it, we run the tests only for java 11, we probably should run it also for java 17

Reply:
The unit tests don't need `protobuf` or any other of the external dependencues, but they need now `SSL` because the ScriptOptions of the Java language container use `SSL` to calculate the `MD5` checksum of imported scripts.

I created https://github.com/exasol/script-languages-release/issues/1094 to fix this and get rid of the dependency later.

Reply:
Changed it to run the unit tests now on Java11 + Java17